ruby_on_rails/linting_and_automatic_check.md

```diff
@@ -90,14 +90,11 @@ eslint --init (Use a popular style guide -> Airbnb)
 then extend the `bin/check` script with:
 
 ```
-echo "\n== ESLint =="
-yarn eslint app/assets/javascripts/**/*.es6
-if [ $? -ne 0 ]; then
-  echo 'ESLint detected issues.'
-  exit 1
-fi
+yarn eslint app/webpack spec/javascripts
```

Comment:
if you do just this without exiting, then your `bin/check` may return 0 and CI will be green

Reply:
I'd propose using `set -e` instead of exiting explicitly

Reply:
the `-e` option is already in place. that's why I changed this part.